### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by anqi20**

### CODE REVIEW

File: docs/team/anqi20.md [hunk 1]
```diff
@@ -4,22 +4,33 @@
...
 
 - **New Feature:** Implemented the timer function for the quiz
   - What it does: The user would be able to choose the timer set for each question. If the time is up, and the user have yet to complete the question, then it will be marked as an incorrect answer. 
-  - Justification: E-Duke-8 is able to simulate a timed examination environment for the students. 
-- **New Feature:** Added the need for users to press "Enter" before they can proceed to the next question or end the quiz
-  - What it does: 
-  - Justification: 
+  - Justification: E-Duke-8 is able to simulate a timed examination environment for the students. Moreover, since they get to choose the timing , E-Duke-8 is able to cater to all students at different learning stages. 
+  - Highlights: This timer feature is challenging as we had to cater to users using different operating systems. Kingston has helped in implementing this feature for the Windows Operating System. 
+- **New Feature:** Added delay between each question during the quiz
```
Comment (kstonekuan) [inline docs/team/anqi20.md:15]: Looks good. I believe you also worked on Ui and some parts of the Model, do you want to mention those as well?

File: docs/team/anqi20.md [hunk 2]
```diff
@@ -4,22 +4,33 @@
...
+## Summary of Contributions
 
 - **New Feature:** Implemented the timer function for the quiz
   - What it does: The user would be able to choose the timer set for each question. If the time is up, and the user have yet to complete the question, then it will be marked as an incorrect answer. 
-  - Justification: E-Duke-8 is able to simulate a timed examination environment for the students. 
-- **New Feature:** Added the need for users to press "Enter" before they can proceed to the next question or end the quiz
-  - What it does: 
-  - Justification: 
+  - Justification: E-Duke-8 is able to simulate a timed examination environment for the students. Moreover, since they get to choose the timing , E-Duke-8 is able to cater to all students at different learning stages. 
+  - Highlights: This timer feature is challenging as we had to cater to users using different operating systems. Kingston has helped in implementing this feature for the Windows Operating System.
```
Comment (kstonekuan) [inline docs/team/anqi20.md]: Feel free to write more detail for this as well!

**Review by kstonekuan — APPROVED** [review]
lgtm

File: docs/team/anqi20.md (hunk 1 above)
Comment (anqi20) [reply docs/team/anqi20.md:15]: Yep, I have updated the PPP as well. Thank you for the comments.

**Pull request merged**